recipe/meta.yaml

```diff
@@ -1,58 +1,60 @@
-{% set version = "0.17.8" %}
+{% set version = "0.17.11" %}
 
 package:
   name: great-expectations
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/great-expectations/great_expectations-{{ version }}.tar.gz
-  sha256: a13bb1827f3c710d059db18cea727961b8fb90d1f7656d03e1c9c5bcf7bb52bf
+  sha256: 484106cd2cd5e2ad19a13ced3283eafa6855822b2bbfed0df545674e6cfac6ea
 
 build:
-  noarch: python
+  skip: True  # [py<36]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - great_expectations = great_expectations.cli:main
 
 requirements:
   host:
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
     - pip
   run:
-    - pydantic >=1.9.2,<2.0
-    - marshmallow >=3.7.1,<4.0.0
-    - makefun >=1.7.0,<2
-    - notebook >=6.4.10
-    - dataclasses
-    - ipython >=7.16.3
+    - altair >=4.2.1,<5.0.0
+    - click >=7.1.2,!=8.1.4
     - colorama >=0.4.3
     - cryptography >=3.2
-    - nbformat >=5.0
-    - packaging
-    - typing-extensions >=3.10.0.0
-    - urllib3 >=1.26
-    - python >=3.6
-    - altair >=4.2.1,<5.0.0
-    - click >=7.1.2,<=8.1.3
-    - importlib-metadata >=1.7.0
+    - ipython >=7.16.3
     - ipywidgets >=7.5.1
     - jinja2 >=2.10
     - jsonpatch >=1.22
     - jsonschema >=2.5.1
+    - makefun >=1.7.0,<2
     - mistune >=0.8.4
-    - numpy >=1.19.5
-    - pandas >=1.1.0,<2
+    - marshmallow >=3.7.1,<4.0.0
+    - nbformat >=5.0
+    - notebook >=6.4.10
+    - numpy >=1.20.3  # [py==38]
+    - numpy >=1.21.6  # [py==39]
+    - numpy >=1.22.4  # [py>=310]
+    - packaging
+    - pandas >=1.1.0  # [py==38]
+    - pandas >=1.1.3  # [py==39]
+    - pandas >=1.3.0  # [py>=310]
+    - pydantic >=1.9.2,<2.0
     - pyparsing >=2.4
     - python-dateutil >=2.8.1
     - pytz >=2021.3
     - requests >=2.20
     - ruamel.yaml >=0.16,<0.17.18
     - scipy >=1.6.0
-    - termcolor >=1.1.0,<2.1.0
     - tqdm >=4.59.0
+    - typing-extensions >=3.10.0.0
     - tzlocal >=1.2
-    - typing_extensions >=3.10.0.0
+    - urllib3 >=1.26
+    - python
   run_constrained:
     - sqlalchemy >=1.3.16
     - pyspark >=2.3.2
```

Comment:
Could you add `great_expectations --help` to `test/commands` as there are `entry_points`